.travis.yml

```diff
@@ -6,9 +6,16 @@ jdk:
 env:
   - UI_DIR=ui
 
-before_script:
-  - cd $UI_DIR && npm install -g grunt-cli bower && npm install && bower install && grunt build
+before_install:
+  - export CHROME_BIN=chromium-browser
+  - export DISPLAY=:99.0
```

Comment:
Can you comment in the config where the setup came from? Thanks for adding it to the PR, but it will be hard to find later.